### ISSUE TITLE
Qo` ko` analysis

### ISSUE THREAD

**Issue by koghuzhan** [apertium/apertium-uzb#2]
The machine seems to break down words such as 

> ko`rsatadi
> qo`mitasi

at **`**.

Maybe it's only when the character used is one other than the actual Ozbek one but it still needs to be dealt with.

**Comment by ftyers** [comment]
> The machine seems to break down words such as
> 
> > ko`rsatadi qo`mitasi
> 
> at **`**.
> 
> Maybe it's only when the character used is one other than the actual Ozbek one but it still needs to be dealt with.

This should be in the spellrelax file. Something like:

```
[
 [ ?* [ ʼ  -> [ %' | ’ | %` | %´ | %‘ | %ʻ ] | ʼ ] ?* ]
]
```

**Comment by ftyers** [comment]
Fixed in b9203d5.

**Comment by ftyers** [comment]
```
$ echo "qo\`mitasi" | apertium -d . uzb-morph
^qo`mitasi/qoʻmita<n><px3sp><nom>/qoʻmita<n><px3sp><nom>+e<cop><aor><p3><pl>/qoʻmita<n><px3sp><nom>+e<cop><aor><p3><sg>$^./.<sent>$
```
